pkg/kubectl/util/BUILD

```diff
@@ -25,7 +25,10 @@ filegroup(
 
 filegroup(
     name = "all-srcs",
-    srcs = [":package-srcs"],
+    srcs = [
+        ":package-srcs",
+        "//pkg/kubectl/util/term:all-srcs",
```

Comment:
This will conflict with your other pull-requests in flight, but I guess that's ok